CHANGES.rst

```diff
@@ -22,6 +22,8 @@ Specviz2d
 API Changes
 -----------
 
+- Subset Plugin now respects the chosen display unit after using Unit Conversion. [#2195]
+
```

Comment:
probably should be removed from the changelog so there is a single entry when merging this into main

Reply:
did this get removed (@rosteen)?

Reply:
Ugh, I did it in my previous squash attempt but not when I redid it. I'll open a PR.